crt_portal/cts_forms/templates/landing.html

```diff
@@ -170,11 +174,9 @@ <h2 class="h1__display text__reverse" id="your-rights">{% trans "Understanding y
     <div class="grid-container">
       <div class="grid-row grid-gap">
         <div class="tablet:grid-col-12">
-          <h2 class="h2__display">{% trans "Common civil rights violations" %}</h2>
-          <div class="crt-landing--separator_small"></div>
           <div class="grid-row grid-gap">
             <div class="tablet:grid-col-7">
-              <p class="crt-landing--largetext">{% trans "Choose from this list to see examples." %}</p>
+              <h2 class="h2__display f">{% trans "Choose from this list to see example civil rights violations:" %}</h2>
```

Comment:
stray `f` here?

Reply:
Oops! Thanks for catching this.
```suggestion
              <h2 class="h2__display">{% trans "Choose from this list to see example civil rights violations:" %}</h2>
```